src/content/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration.mdx

```diff
@@ -1298,9 +1298,45 @@ Available logging-related config options include:
 
 </CollapserGroup>
 
-## Code Level Metrics
+## Set version tag [#version-tag]
+
+Setting NEW_RELIC_METADATA_SERVICE_VERSION will create a tag, tag.service.version on event data. Service.version is the version of your code that is deployed, in many cases a semantic version such as 1.2.3 but not always. Sending this information allows you to facet your telemetry by the version of the software deployed so you can quickly identify which versions of your software are producing the errors. 
```

Comment:
I think it's a little odd to have the second sentence start with Service.version since case for field names matters.  Perhaps you could start the sentence with something like: "The attribute, `tags.service.version` is...."   I think this might be an issue on all the config pages.

Reply:
Did you double major in English? Thank you for the suggestions. I was struggling with the introductory phrase. I will go ahead and make the update with "In this context," for this and all the other agent updates :)